pyne/mesh.py

```diff
@@ -1140,6 +1181,23 @@ def _do_op(self, other, tags, op, in_place=True):
                 #removed the [0] from the ops call
                 mesh_1.mesh.tag_set_data(mesh_1_tag, ve_1, val)
         return mesh_1
+    
+    def multiply_vector_tag_by_scalar(self, vector_tag_name, scalar):
```

Comment:
I'm not sure we need this method anymore?  We have discovered better ways to accomplish this capability already in  the interface.